subprojects/dependency-management/src/testFixtures/groovy/org/gradle/integtests/fixtures/publish/RemoteRepositorySpec.groovy

```diff
@@ -76,6 +76,19 @@ class RemoteRepositorySpec {
         }
     }
 
+    def gav(String notation, @DelegatesTo(value=ModuleVersionSpec, strategy = Closure.DELEGATE_ONLY) Closure spec = {}) {
```

Comment:
Looking for some feedback here. This is used for the `repository` and `repositoryInteractions` blocks in `AbstractModuleDependencyResolveTest`s. Its current usage relies on `methodMissing`, making IDE integrations/hints impossible. However, using this method does allow us to use IDE hints. 

My initial thoughts on naming was `module`, except the block already supports a method with that name by using the following syntax:
```
repository {
    group('group') {
        module('module) {
            // ...
        }
    }
}
```
So I thought a top-level `module` method would be confusing. `gav` seems pretty straightforward, though I'm not sure how tied we are to that nomenclature when referring to a ModuleVersionIdentifier.

Reply:
Switched to `id` for better future overloading possibilities.